queue/abstract.lua

```diff
@@ -495,6 +532,31 @@ function method.create_tube(tube_name, tube_type, opts)
     return self
 end
 
+--- Create everything that's needed to work with "shared" sessions.
+local function identification_init()
```

Comment:
As raw idea: will the code looks better, if we'll move all session management stuff to its own file like queue/session.lua?

Reply:
More I look into the code, more promising the idea looks for me. When I'm looking into queue code, I want to concentrate on the data layout, ordering guarantees, wakeups and other queue logic. Not on session management.

Reply:
Done.